packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -326,8 +326,9 @@ export class DumbTriggerManager extends Component {
     const showSpinner = submitting && selectedCipher && step === 'ciphersList'
     const showCiphersList = step === 'ciphersList'
     const showAccountForm = step === 'accountForm'
+    const konnectorPolicy = findKonnectorPolicy(konnector)
 
-    if (oauth) {
+    if (oauth || konnectorPolicy.isWebView) {
```

Comment:
I didn't see that during my first review, but can we rename isWebView to `isBIWebView`? Since our CCC are in "webview", I'm sure it'll avoid confusion in the future